bin/ch/WScriptJsrt.cpp

```diff
@@ -37,6 +38,8 @@
 
 MessageQueue* WScriptJsrt::messageQueue = nullptr;
 std::map<std::string, JsModuleRecord>  WScriptJsrt::moduleRecordMap;
+std::map<JsModuleRecord, std::string>  WScriptJsrt::moduleDirMap;
```

Comment:
it is fine for limited current use for now, but this will not work when running with multiple threads.
Also, JsModuleRecord is Js::SourceTexttMdouleRecord. with is GC allocated.
That means that if the JsModuleRecord is collected, you will have dangling records here.